src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -4499,85 +4503,84 @@ namespace ZoneEquipmentManager {
         // Sensible output updates
         auto &thisZEqList(DataZoneEquipment::ZoneEquipList(ctrlZoneNum));
         switch (thisZEqList.LoadDistScheme) {
-        case DataZoneEquipment::LoadDist::SequentialLoading:
```

Comment:
I didn't change anything here. Certainly don't merge this if it changes anything.

Reply:
Looks like Clang just pulled up the open bracket and moved every line within that bracket left to realign the code. So this looks OK, except the Clang is flaky. Can anyone proficient with Clang tell us why this keeps happening?

Reply:
I don't think it's being flaky, just fixing formatting that wasn't proper before. In this particular spot, that extra set of braces embracing the case block isn't necessary. You can either live with these changes as they come in, or turn off auto formatting in VS and just manually format selected sections that you've worked on.